frontend/views/containers/chatroom/SendArea.vue

```diff
@@ -376,9 +376,9 @@ export default ({
   },
   created () {
     // TODO #492 create a global Vue Responsive just for media queries.
-    const mediaIsPhone = window.matchMedia('(hover: none) and (pointer: coarse)')
-    this.ephemeral.isPhone = mediaIsPhone.matches
-    mediaIsPhone.onchange = (e) => { this.ephemeral.isPhone = e.matches }
+    this.mediaIsPhone = window.matchMedia('(hover: none) and (pointer: coarse)')
```

Comment:
This seems to be creating a variable on the Vue object on-the-fly, which, as far as I know, isn't recommended by Vue.js. So correct me if I'm wrong, but I think this `mediaIsPhone` should be defined in the object returned by the `data()` section.

Reply:
Fixed it.